privacy-config/privacy-config-api/build.gradle

```diff
@@ -31,6 +31,7 @@ kotlin {
 }
 
 dependencies {
+    implementation project(path: ':feature-toggles-api')
```

Comment:
Yeah, that's what I was saying before. Maybe we want a `PrivacyConfigException` instead of using the `FeatureException`.
No strong opinions tbh. I think long term we want to migrate everything to use the feature-toggles module